app/controllers/coronavirus_form/need_help_with_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::NeedHelpWithController < ApplicationController
+  def submit
+    @form_responses = {
+      need_help_with: Array(params[:need_help_with]).map { |item| strip_tags(item).presence }.compact,
```

Comment:
It would be super useful if we could store the key of the question groups that we should display to the user based on their answer to this question

Reply:
In @form_responses or in the session?

Reply:
In the session

Reply:
So like:
```
session[:feeling_safe] = true
session[:paying_bills] = true
```
If they selected "Feeling safe" and "Paying bills"?
```

Reply:
Yes, but maybe all under one session key?

E.g.
```
session[:groups_selected] = {
  felling_safe: true,
  paying_bills: true,
}
```

This will make it easier to work out which questions we should show the user.

Reply:
Updated and pushed up

Reply:
Thanks, this will make my current work so much easier